### ISSUE TITLE
"lang" is undefined at docsify-pdf-embed.js:22:7

### ISSUE THREAD

**Issue by Stalker2106x** [lazypanda10117/docsify-pdf-embed#6]
Hello,
I added the correct two lines, and fired up node:
Cannot load the page, I have an error on "lang" variable.
Did i miss something in the configuration ?

Awaiting your answers,
Thank you.

**Comment by lazypanda10117** [comment]
Hi @Stalker2106x , could you provide more information regarding the error? For instance, the browser you used, the lines you added, and actual error messages from console, and perhaps even a link to a screenshot of the behaviour. It is hard to pinpoint the cause of problem given only that. Thanks.

**Comment by Stalker2106x** [comment]
You're right, I just thought it is a common mistake as I just injected the getting started in my project.
My config for docsify is the following:
`{
      name: 'Documentation',
      repo: '',
      logo: './img/logo.png',
      maxLevel: 6,
      subMaxLevel: 6,
      fallbackLanguages: ['fr'],
      search: {
        depth: 6,
        paths: ['/']
      },
      markdown: {
      }
    }`

I injected the library via:
` <script src="//cdnjs.cloudflare.com/ajax/libs/pdfobject/2.1.1/pdfobject.min.js"></script> 
  <script src="./js/docsify-pdf-embed.js"></script>`

And the lang argument of your plugin callback is always undefined. If I add a default value for lang parameter, plugin just writes "Link to ...pdf" in place of embed thing.

**Comment by lazypanda10117** [comment]
I tried to reproduce the problem but couldn't seem to find an error yet. I will look more into it later this week. In the meantime, if you just want to make it work, you can take a look at my configurations here and see if you can figure something out. 
[Configurations](https://github.com/lazypanda10117/random-notes/blob/master/docs/index.html)
[Example](https://github.com/lazypanda10117/random-notes/blob/master/docs/PMATH348/notes/Main%20Notes.md)

Note: Not sure if you did, but in case you didn't, you have to put those lines
```javascript
  <script src="//cdnjs.cloudflare.com/ajax/libs/pdfobject/2.1.1/pdfobject.min.js"></script> 
  <script src="./js/docsify-pdf-embed.js"></script>
```
after the docsify script import statement
```javascript
  <script src="//unpkg.com/docsify/lib/docsify.min.js"></script>
```

**Comment by lazypanda10117** [comment]
Sorry, there was not enough context previously for me to determine what the exact problem is. However, I think the problem is now resolved as a part of a fix for issue #8 . Thanks!
